packages/components/data-table/src/data-table.story.js

```diff
@@ -403,7 +403,9 @@ storiesOf('Components|DataTable', module)
               : null
           }
           maxHeight={castToNumberIfPossible(
-            text('maxHeight', 'calc(100vh - 200px)')
+            // in this example, the table has a dynamic height equal to the viewport height minus 200px
+            // floored to a minimum value of 400px
+            text('maxHeight', 'max(400px, calc(100vh - 200px))')
```

Comment:
Unrelated, but added this just to improve the quality of the example and show what's possible to do.